src/lib/connect/index.ts

```diff
@@ -152,6 +133,28 @@ function connect(
 		}
 	}
 
+	const protocols: Record<string, StreamBuilder> = {}
+
+	if (!opts.forceNativeWebSocket && !isBrowser) {
+		protocols.ws = require('./ws').streamBuilder
+		protocols.wss = require('./ws').streamBuilder
+
+		protocols.mqtt = require('./tcp').default
+		protocols.tcp = require('./tcp').default
+		protocols.ssl = require('./tls').default
+		protocols.tls = protocols.ssl
+		protocols.mqtts = require('./tls').default
+	} else {
+		protocols.ws = require('./ws').browserStreamBuilder
+		protocols.wss = require('./ws').browserStreamBuilder
+
+		protocols.wx = require('./wx').default
+		protocols.wxs = require('./wx').default
+
+		protocols.ali = require('./ali').default
+		protocols.alis = require('./ali').default
+	}
```

Comment:
I would do this only once. Keep protocols outside the function and put it to null by default, if null load the protocols, skip otherwise

Reply:
I don't think there could be any valid reason to change that option when creating different clients, do you?

Reply:
No indeed, but this could be confusing to have an object option influencing the global space. I will make sure to document that in the README then.